buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -56,12 +52,11 @@ source activate $CONDA_ENV
 set -v
 
 # Install optional packages into activated env
-
 echo "PYTHON=$PYTHON"
 echo "VANILLA_INSTALL=$VANILLA_INSTALL"
-if [[ "$PYTHON" != "3.10" && "$VANILLA_INSTALL" != "yes" ]]; then
-    # Scipy, CFFI, jinja2, IPython, and pygments are optional dependencies,
-    # but exercised in the test suite.
+if [ "${VANILLA_INSTALL}" != "yes" ]; then
+    # Scipy, CFFI, jinja2, IPython, ipython_genutils and pygments are optional
```

Comment:
I wonder if I should add scipy back in here, it doesn't seem to be installed at all at present. Also, perhaps I should remove `ipython_genutils`?

Reply:
Looks like SciPy is conditional on "not 32 bit linux"?
https://github.com/numba/numba/blob/73e84b50bd8a80980f0349528002fee7d653b014/buildscripts/incremental/setup_conda_environment.sh#L68-L72 ?

Reply:
RE: `ipython_genutils`. I think what is in `main` is correct:
https://github.com/numba/numba/blob/73e84b50bd8a80980f0349528002fee7d653b014/buildscripts/incremental/setup_conda_environment.sh#L63-L66

Reply:
Yes, it makes sense. I will remove `ipython_genutils` from the comment.